helm-charts/splunk-otel-collector/templates/configmap-gateway.yaml

```diff
@@ -1,5 +1,5 @@
 {{ $gateway := fromYaml (include "splunk-otel-collector.gateway" .) }}
-{{ if $gateway.enabled }}
+{{ if or $gateway.enabled (eq (include "splunk-otel-collector.distribution" .) "eks/fargate") }}
```

Comment:
Looks like we don't install gateway by default, so we shouldn't install gateway config either.